packages/my-react-package/src/constants/chains.ts

```diff
@@ -0,0 +1,6 @@
+export enum SupportedChainId {
```

Comment:
You have specified a list of supported networks twice. Once here and another time inside the MetaContract. 
Modify the code to have this file as the source of truth.